src/Cms/Language.php

```diff
@@ -288,6 +295,14 @@ public function isLast(): bool
 		return App::instance()->languages()->count() === 1;
 	}
 
+	/**
+	 * Checks if this is the single language object
```

Comment:
As this will change with the new always-multilang concept, I think we should make this internal:

```suggestion
	 * Checks if this is the single language object
	 * @internal
```